### PR TITLE
[polaris.shopify.com] Responsive examples

### PR DESCRIPTION
**Pull request by kyledurand**
https://user-images.githubusercontent.com/6844391/187300794-76e7f2e5-10dc-48c6-9e2c-8c41307412d3.mp4

### CODE REVIEW

**Comment by martenbjork** [comment]
I like it! I made some tiny design suggestions: https://github.com/Shopify/polaris/pull/7059

**Comment by martenbjork** [comment]
If I understand things correctly, the problem with the skeleton example is:

- Most examples need to be centered (button, badge) etc. We use flex to center this content.
- When the centered content doesn't have an intrinsic width, flex sets its width to 0
- ...causing the example to be hidden or squished

A while back, I was jamming on a broader, catch-all solution for this. Could it be time to revisit it?

https://github.com/Shopify/polaris/pull/6403

File: .changeset/smart-garlics-kneel.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'polaris.shopify.com': minor
+---
+
+Allowed examples to be responsive, fixed SkeletonBodyText example
```
Comment (kyledurand) [inline .changeset/smart-garlics-kneel.md]:
```suggestion
Allowed examples to be responsive
```

**Comment by kyledurand** [comment]
That sounds right to me @martenbjork. I've removed references to that bug fix from this PR and just made it about responsive examples. Thanks for the slick styling / css cleanup ❤️

**Review by martenbjork — APPROVED** [review]

**Pull request merged**